gripper/firmware/motor_timer_hardware.c

```diff
@@ -10,6 +19,16 @@ TIM_OC_InitTypeDef htim3_sConfigOC = {0};
 static motor_interrupt_callback timer_callback = NULL;
 static brushed_motor_interrupt_callback brushed_timer_callback = NULL;
 
+uint32_t clamp(uint32_t val, uint32_t min, uint32_t max) {
```

Comment:
By `clamp` you mean set to within the bounds of min and max? Not actually actuate the gripper right? 

Dumb question I know.

Reply:
yeah correct. numeric clamp